scripts/install.sh

```diff
@@ -391,10 +391,17 @@ sed_in_place "/.*# avalanche completion/d" $BASH_COMPLETION_MAIN
 echo "source $BASH_COMPLETION_SCRIPT_PATH # avalanche completion" >> $BASH_COMPLETION_MAIN
 if [ $(uname) = Darwin ]
 then
-    BASHRC=~/.bashrc
-    touch $BASHRC
-    sed_in_place "/.*# avalanche completion/d" $BASHRC
-    echo "source $(brew --prefix)/etc/bash_completion # avalanche completion" >> $BASHRC
+    BREW_INSTALLED=true
+    which brew > /dev/null 2> /dev/null || BREW_INSTALLED=false
+    if [ $BREW_INSTALLED = true ]
+    then
+        BASHRC=~/.bashrc
```

Comment:
If we are generally limiting scripts to `bash` only, I guess this is fine. Obviously this wouldn't work for `zsh`. I don't know if we have any such limitation though.

Reply:
the brew dependency is related to bash configuration

Reply:
? 
I don't understand this comment.
Either we support both bash and zsh or only bash, and it seems we do both?